packages/@sanity/form-builder/src/__workshop__/_common/input.tsx

```diff
@@ -0,0 +1,153 @@
+import React, {
```

Comment:
Not sure if the naming of this file (`input.tsx`) makes all that much sense to me. Could we try to come up with a better name? Alternatively would it make sense to split it into different component files?

Reply:
I've split it up into separate component files and renamed input to `DebugInput`